drivers/usb/device/usb_dc_stm32.c

```diff
@@ -375,6 +382,26 @@ static int usb_dc_stm32_init(void)
 	usb_dc_stm32_state.pcd.Init.Sof_enable = 1;
 #endif /* CONFIG_USB_DEVICE_SOF */
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+	/* Currently assuming FS mode. Need to disable the ULPI clock on USB2 and
+	 * enable the FS clock. Need to make this dependent on HS or FS config.
+	 */
+
+	__HAL_RCC_USB2_OTG_FS_ULPI_CLK_DISABLE();
+	__HAL_RCC_USB2_OTG_FS_ULPI_CLK_SLEEP_DISABLE();
+	__HAL_RCC_USB2_OTG_FS_CLK_ENABLE();
+	__HAL_RCC_USB2_OTG_FS_CLK_SLEEP_ENABLE();
```

Comment:
Please use LL API

Reply:
I've switched everything to use the LL API.